### PR TITLE
logic_update_profile

### PR DESCRIPTION
**Pull request by datToc004**
## Related Tickets:
- [#40316](https://edu-redmine.sun-asterisk.vn/issues/40316)

## WHAT this PR does?

logic update profile
## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by datToc004** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?php
 
+use App\Http\Controllers\frontend\HomeController;
```
[the remark this answers is not in the document]
Comment (manhtd-1083) [reply routes/web.php:3]: chưa fix comment nè

[the remark this answers is not in the document]
Comment (datToc004) [reply routes/web.php:3]: sao lạ vậy! nhỉ chắc quên đấy!

**Comment by datToc004** [comment]
ready

**Comment by datToc004** [comment]
ready

File: app/Http/Controllers/Customer/HomeController.php [hunk 2]
```diff
@@ -27,4 +27,18 @@ public function dish()
...
     }
+
+    public function profile()
+    {
+        return view('frontend.profile.profile');
+    }
+
+    public function postProfile(ProfileRequest $request)
+    {
+        $user = User::findOrFail(Auth::user()->id);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Customer/HomeController.php:38]:
```suggestion
        $user = Auth::user();
```

File: app/Http/Requests/ProfileRequest.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        return [
+            'email.required' => 'email không được để trống',
+            'email.email' => 'email chưa nhập đúng',
+            'email.unique' => 'email đã trùng lặp',
+            'name.required' => 'name không được để trống',
+            'phone.required' => 'phone không được để trống',
+            'sex.required' => 'sex không được để trống',
+            'address.required' => 'address không được để trống',
+        ];
```
Comment (huyenttm-0901) [inline app/Http/Requests/ProfileRequest.php:45]: i18n

**Comment by datToc004** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**